### PR TITLE
Improve notes_t wording

### PR DESCRIPTION
**Pull request by tschmidtb51**
- resolves oasis-tcs/csaf#546
- modify wording in notes_t for consistency and clarity

### CODE REVIEW

**Review by santosomar — APPROVED** [review]
These grammar changes are correct. A reference related to the change about "content" vs "contents": https://dictionary.cambridge.org/us/grammar/british-grammar/content-or-contents

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 1]
```diff
@@ -1251,7 +1251,7 @@ Audience of note (`audience`) of value type `string` with 1 or more characters i
     safety engineers
 ```
 
-Note category (`category`) of value type `string` and `enum` indicates the choice of what kind of note this is.
+Note category (`category`) of value type `string` and `enum` contains the information of what kind of note this is.
```
Comment (sthagen) [inline csaf_2.0/prose/csaf-v2-editor-draft.md:1254]: Nit: or, simply ˋnames the category the note belongs tò?

**Review by sthagen — APPROVED** [review]

File: csaf_2.0/prose/csaf-v2-editor-draft.md (hunk 1 above)
Comment (tschmidtb51) [reply csaf_2.0/prose/csaf-v2-editor-draft.md:1254]: I tried to avoid the "word is explained by itself"...

Comment (sthagen) [reply csaf_2.0/prose/csaf-v2-editor-draft.md:1254]: Sometimes words are chosen for a reason :wink: - "containing information of  what kind" to me blurs what the well-chosen term `category` achieved, but I am OK either way

**Comment by santosomar** [comment]
This pull request was reviewed and approved by the TC during the OASIS CSAF TC Meeting on May 18, 2022. The proposed changes are non-material and do not affect the schema. The contributions enhance consistency and provide additional clarity to implementers.

**Pull request merged**